### PR TITLE
Update native-shim.js to not use new.target

### PR DESCRIPTION
**Pull request by justinfagnani**
Do not merge. For discussion.

### CODE REVIEW

File: src/native-shim.js [hunk 1]
```diff
@@ -21,10 +21,7 @@
   let origHTMLElement = HTMLElement;
   // TODO(justinfagnani): Tests!!
   window.HTMLElement = function() {
-    // prefer new.target for elements that call super() constructors or
-    // Reflect.construct directly
-    let newTarget = new.target || this.constructor;
-    return Reflect.construct(origHTMLElement, [], newTarget);
+    return Reflect.construct(origHTMLElement, [], this.constructor);
   }
```
Comment (JanMiksovsky) [inline src/native-shim.js]: I'm seeing a lint error complaining that this is an assignment, so a `;` after the closing brace is called for.

**Comment by JanMiksovsky** [comment]
Before commiting, the comments about `new.target` should be removed.

**Comment by snuggs** [comment]
#ping can we close this @justinfagnani @JanMiksovsky ?

**Comment by justinfagnani** [comment]
Yeah, thanks @snuggs

**Pull request closed without merging**